src/hatti/views/table.cljs

```diff
@@ -71,7 +71,9 @@
           query (aget args "query")
           fmt-subitem (fn [[fname answer]]
                         (format-answer (get indexed-form fname)
-                                       answer nil true))
+                                       answer
+                                       :language nil
```

Comment:
no reason to explicitly pass this as `nil`, that's what it will be when not passed